project/settings.py

```diff
@@ -471,39 +471,41 @@
 CELERY_RESULT_BACKEND = 'djcelery.backends.database:DatabaseBackend'
 
 # Celery Queues
-
-DEFAULT_QUEUE = 'celery'
-LOW_QUEUE = 'low'
-MED_QUEUE = 'med'
-HIGH_QUEUE = 'high'
-
-LOW_PRI_MODULES = {
-    'share.tasks.HarvesterTask',
-}
-
-# Default priority is implicit
-# DEFAULT_PRI_MODULES = {
-#     'share.tasks.NormalizerTask',
-# }
-
-MED_PRI_MODULES = {
-    'share.tasks.DisambiguatorTask',
-}
-
-HIGH_PRI_MODULES = {
-    'share.tasks.BotTask',
-    'bots.elasticsearch',
+QUEUES = {
+    'DEFAULT': {
+        'name': 'celery',
+        'priority': 0,
+        'modules': set(),
+    },
+    'GEVENT': {
```

Comment:
Should this queue only exist depending on the `GEVENT` env variable?  Or does it make no difference